Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.access.redhat.com/ubi9-minimal:latest
```

Comment:
Just curious, why this instead of `registry.access.redhat.com/ubi9/ubi-minimal`?

I'd love to know why both [ubi9/ubi-minimal](https://catalog.redhat.com/software/containers/ubi9/ubi-minimal/615bd9b4075b022acc111bf5?container-tabs=gti&gti-tabs=unauthenticated) and [ubi9-minimal](https://catalog.redhat.com/software/containers/ubi9-minimal/61832888c0d15aff4912fe0d?container-tabs=gti&gti-tabs=unauthenticated) exist.

Reply:
I just went off what I could find when searching https://catalog.redhat.com/

Reply:
Cool deal. It's weird they both exist, but 🤷🏻